data/bountive.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Bountive",
+  "description": "Bountive is the first decentralized prize savings protocol on Starknet, empowering users with no-risk prize games. Bountive is based on the proven concept known as “Prize Linked Savings Accounts”. This product helps to boost savings. Now, we’re bringing this to the blockchain.",
+  "short_description": "An innovative solution to save & invest your digital assets",
```

Comment:
@Hug0x0 The short description is only shown on the listing pages, so should be really short, please, e.g. "No risk prize savings games"